dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/IconService.java

```diff
@@ -29,6 +29,7 @@
 
 import java.sql.SQLException;
 import java.util.Set;
+import lombok.NonNull;
```

Comment:
that is the wrong one see https://github.com/dhis2/wow-backend/blob/59d3a443c755080a0da845251ca5db2d07dcd4a0/guides/preferred_patterns.md?plain=1#L27

Reply:
i am using `javax.annotation`.